instrumentation/jms/src/main/java/brave/jms/PropertyFilter.java

```diff
@@ -74,6 +80,31 @@ static void filterProperties(Message message, Set<String> namesToClear, List<Obj
       return;
     }
 
+    // workaround for ActiveMQBytesMessage
+    if (message instanceof BytesMessage) {
```

Comment:
I see this check being done here too. Worth to put into a private method that selfdescribes the why?

Reply:
Please help me to get rid of the other usage. I believe instanceof is simple enough thought.

Reply:
Yeah, it is. so maybe a comment explaining the why is enough.

Reply:
Lets add that comment to this method with why there is a special case for `BytesMessage`

Reply:
I added a comment to resetBytesMessageProperties

Reply:
I don't wanna be a pain but I'd really prefer a comment in the `instanceof` check. It is way easier to read rather than having go to to the function to understand the check.

Reply:
fixed